js/utils.js

```diff
@@ -183,11 +183,16 @@ Fliplet.Registry.set('dynamicListUtils', function() {
 
     return _.filter(records, function(record) {
       return _.every(filters, function(filter) {
+        if (!filter.condition || !filter.value || filter.column === 'none') {
```

Comment:
what if one of these is `undefined`, can it happen? e.g. `filters [ undefined, null ]`

Reply:
It can't. The default is created as `"none"` (just pushed a commit to match this), and the `<select>` has a default value of `"none'` as well. I don't like it and would've preferred `null` but I'm not risking changing this.